app/core/search.py

```diff
@@ -1231,8 +1231,17 @@ def process_vespa_search_response(
     vespa_search_response: DataAccessSearchResponse,
     limit: int,
     offset: int,
+    document_extra_info: Mapping[str, Mapping[str, str]],
 ) -> SearchResponse:
     """Process a Vespa search response into a F/E search response"""
+
```

Comment:
Should this logic be on line 1118, or should we remove that comment and debug line?

Reply:
Do you mean in this function? 

`_process_vespa_search_response_families`

The reason I didn't put it in there is just so we could get the hits count. This obviously isn't a blocker however so I'm happy to refactor!